theme/static/style/style.css

```diff
@@ -412,6 +456,72 @@ pre code {
 @media screen and (max-width: 991px){
     /* For tablets: */
 
+    /*navigation section*/
+    .topnav .col-4 img{
+        margin-top: 1.25rem;
+        max-width: 20rem;
+    }
+
+    .topnav .col-8 li:last-child{
+        display: block;
+    }
+
+    .topnav .open-menu{
+        display: inline-block;
+        width: 5rem;
+        height: 5rem;
+        border-radius: 0.3rem;
+        border: 0;
+        cursor: pointer;
+        margin-right: 1.5rem;
+        float: right;
+    }
+
+    .mobilenav{
+        position: fixed;
+        top: -100%;
+        left: 0;
+        width: 100%;
+        height: 100%;
+        z-index: 2;
+        transition: .5s top ease;
+        overflow-y: scroll;
+    }
+
+    .mobilenav  .close-menu{
+        position: absolute;
+        top: 1rem;
+        left: 1rem;
+        display: block;
+        width: 5rem;
+        height: 5rem;
+        border-radius: 0.3rem;
+        border: 0;
+        cursor: pointer;
+    }
+
```

Comment:
All of the `.mobilenav`-something selectors below can be converted to `ul.topnav`-something after the revert and I think some of the properties even deleted because they seem to be duplicating the already existing definitions.

NOTE: after I did the proposed changes the menu was working for me but the list didn't change to display one item on a row. Once you have less changes in the entire PR we can figure how to remedy this though.